standards/react_native_standards.md

```diff
@@ -0,0 +1,121 @@
+  > `const variable = 'test';` <br>
+  > `let variableBoolean = true;`
+
+- Use `.jsx` or `.tsx` extension a for React components
+
+## Files and Folders Structure
+
+###  Grouping by Features or Navigation
+- One of the common approach to structure projects is to locate all main screens in one folder, all reusable components in one folder and style related files in one folder grouped by feature or navigation.
+
+###  Grouping by File Type
+- Another popular approach to structure projects is to group similar files together.
+
+## Importing Modules
+
+###  Import Grouping
+- Standard Libraries (React, React-Native)
+
+  ```csharp
+  import React from 'react';
+  import { TouchableOpacity, View } from 'react-native';
+  ```
+
+- Third-Party Libraries
+
+  ```csharp
+  import axios from 'axios';
+  import lodash from 'lodash';
+  ```
+
+- Your Own Code Imports
+
+  ```csharp
+  import { Button, Card } from '../components';
+  import { MainLayout } from '../layouts';
+  ```
+
+- Module-Specific Imports (styles, PNG, etc.)
+
+  ```csharp
+  import styles from'./myComponent.styles';
+  import image from './image.png';
+  ```
+
+- Test-Only Imports
+
+  ```csharp
+  import { testFunction } from './tests';
+  ```
+
+### Sorting
+
+- **Alphabetical Order**: Enforcing alphabetical order can make it easier to locate imports quickly. This could be applied within each import group.
+
+- **Destructured Imports**: If the group contains only a few items, breaking them to one item per line may enhance readability.
+
+- **ESLint or Prettier Rules**: Use linting tools like ESLint or code formatters like Prettier to automate and enforce coding standards. These tools often have rules or configurations for sorting and formatting imports.
+
+## Writing with TypeScript
+
+- To improve code readability, TypeScript comes into play. TypeScript adds types to JavaScript, enabling us to specify the type of each property in the object, reducing the possibility of errors. With TypeScript, you and your editor are aware of the object properties and their types in the code, making it easier to understand and extend the functionality.
+
+## Path Aliases
+
+- Using path aliases is giving your code a shortcut to make import paths shorter and more understandable. This is especially handy when you have many folders nested inside each other, making your imports easier to read.
+
+## Usage of Double Quotes (" ") & Single Quotes (' ')
+- To avoide the confusion about double quotes (" ") & single quotes (' '), use double quotes (" ") for JSX attributes and single quotes (' ') for the JS code
+
+## Crash Reporting Tools
+
+- Crash analytics tools are like superhero sidekicks for your app, always watching for any trouble 24/7. They quickly catch and understand app crashes and errors in real-time, helping you figure out what went wrong.
+- Some cool crash analytics tools are `Sentry`, `Firebase`, `Crashlytics`, and more
```

Comment:
Let's sync up with the team what's our default standard for this, I know it's Firebase Crashlytics